src/components/explorer/IndividualOverview.js

```diff
@@ -49,19 +45,21 @@ IndividualOverview.propTypes = {
 };
 
 function getAge(individual) {
-    // This isn't a real Phenopackets value like UNKNOWN_SEX is
```

Comment:
is this comment still relevant? the `UNKNOWN_AGE` value looks like a real value, so maybe if it isn't a true age value we should use `EM_DASH` like elsewhere

Reply:
Function was removed in favor of TimeElement component, returns an `EM_DASH` if no `time_at_last_encounter` is present